feature/networkinstance/policybasedvrfselection/ate_tests/protocol_dscp_rules_for_vrf_selection_test/protocol_dscp_rules_for_vrf_selection_test.go

```diff
@@ -32,14 +32,18 @@ import (
 const (
 	ipv4PrefixLen = 30
 	ipv6PrefixLen = 126
-	instance      = "default"
 )
 
 // testArgs holds the objects needed by a test case.
 type testArgs struct {
-	dut *ondatra.DUTDevice
-	ate *ondatra.ATEDevice
-	top *ondatra.ATETopology
+	dut                    *ondatra.DUTDevice
+	ate                    *ondatra.ATEDevice
+	top                    *ondatra.ATETopology
+	srcEndPoint            *ondatra.Interface
+	policyName             string
+	iptype                 string
+	protocol               oc.E_PacketMatchTypes_IP_PROTOCOL
+	defaultNetworkInstance string
```

Comment:
If "defaultNetworkInstance string" is referring to *deviations.DefaultNetworkInstance, we can removed this from args list?
If it is non default instance, may be we can add variable something like dutNetworkInstance

Reply:
done